ic-os/setupos/data/deployment.json.template

```diff
@@ -1,6 +1,7 @@
 {
   "deployment": {
-    "name": "mainnet"
+    "name": "mainnet",
+    "mgmt_mac": null
```

Comment:
I added the mgmt_mac field to deployment because doing so was easier for the parsing backwards compatibility than creating a new "network" containing key, but I can add a network key if that feels more proper

Reply:
I think this should be fine since the field is only used in tests, but even changes like this can bite us with backwards compatibility. Once we move the field, our repo tools won't be able to modify old versions of images. We're not often testing too far back in time, though, and it should be easy enough to modify directly when needed.